scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
I updated the code in commit cf1a3af9. The new code looks nicer:

```fish
for plugin in (kdb list)
  kdb info $plugin provides ^/dev/null | string match -qr -- storage
  and echo $plugin
end
```

. However, the runtime of `__fish_kdb_print_storage_plugins` is even worse than before (~ 200ms). Running `kdb list` takes about 200ms on my machine. Calling `kdb info` for all of the 83 plugins takes about 600ms. Altogether it takes approximately 800ms to display all available storage plugins 😴.